d3a_interface/constants_limits.py

```diff
@@ -41,7 +41,7 @@ class GeneralSettings:
         ENERGY_RATE_DECREASE_PER_UPDATE = 1  # rate decrease in cents_per_update
         SETUP_FILE_PATH = None  # Default path of the available setup files
         EXPORT_DEVICE_PLOTS = True
-        EXPORT_ENERGY_TRADE_PROFILE_HR = False
+        HIGH_RESOLUTION_PLOTS = False
```

Comment:
Not sure about this renaming. There might be more HR plots in the future.
And we explicitly prependet `EXPORT` before.

Reply:
Discussed offline, adding a separate flag for OFFER/BID/TRADE graph.